fs/xfstests.py.data/local.config

```diff
@@ -0,0 +1,4 @@
+export TEST_DEV=/dev/loop0
+export TEST_DIR=/mnt/test
+export SCRATCH_DEV=/dev/loop1
+export SCRATCH_MNT=/mnt/scratch
```

Comment:
yes it need to edit via user as it require valid input (Device) of test trigger , i think both way is fine as we need to have in  config  so i think editing in yaml than dump those entry config seems not good so it is better to edit config file directly  i think this approach looks me ok as xfstest follow this  ,